storybookjs/storybook
# Replace Emotion theme context with a custom context implementation

Issue: https://github.com/storybookjs/storybook/issues/10231#issue-589122617

## What I did
I have added a custom context to the storybook theming package and replaced the previous emotion apis or rewritten it to use the new context.

## How to test
I have started a storybook from the examples directory to check if there are errors in the console. I am open for further tests.

- Is this testable with Jest or Chromatic screenshots? - No
- Does this need a new example in the kitchen sink apps? - No
- Does this need an update to the documentation? - No

## Notes
Please have a look at my code and add improvements if necessary.

## Credits
- Thanks to @ndelangen for the introduction to Storybook
- Thanks to @connor-baer for the helpful code example
